node-require/calculate.js

```diff
@@ -0,0 +1,17 @@
+const add = require('./add');
+const subtract = require('./subtract');
+const multiply = require('./multiply');
+const divide = require('./divide');
+
+let result;
+if (process.argv[3] === 'plus') {
+  result = add(parseInt(process.argv[2], 10), parseInt(process.argv[4], 10));
+} else if (process.argv[3] === 'minus') {
+  result = subtract(parseInt(process.argv[2], 10), parseInt(process.argv[4], 10));
+} else if (process.argv[3] === 'over') {
+  result = divide(parseInt(process.argv[2], 10), parseInt(process.argv[4], 10));
+} else if (process.argv[3] === 'times') {
+  result = multiply(parseInt(process.argv[2], 10), parseInt(process.argv[4], 10));
+}
+
```

Comment:
Since you are accessing `process.argv` in several places and using `parseInt` in several places it would be better and easier to save those values into variables first. For example:
```JAVASCRIPT
const num1 = parseInt(process.argv[2], 10);
const num2 = parseInt(process.argv[4], 10);
const op = process.argv[3];
```

Reply:
That's better! I will edit my code and merge the pull request. Thank you!